seaworthy/_lowlevel.py

```diff
@@ -3,10 +3,17 @@
 didn't have to build and maintain ourselves.
 """
 
+import socket
 import struct
 import time
-# selectors is a stdlib module, but flake8-import-order doesn't know that.
-import selectors
+from contextlib import contextmanager
+
+from urllib3.exceptions import ReadTimeoutError
+
+
+# We want time.monotonic on Pythons that have it, otherwise time.time will have
+# to do.
+get_time = getattr(time, 'monotonic', time.time)
```

Comment:
Did this become a problem? When is `time.monotonic()` not available?

Reply:
Various other code I was looking at does it that way and since it's pretty trivial, I thought I may as well. The docs for `time.monotonic()` say "new in 3.3, always available in 3.5" but don't really shed much light on when it might not be available.